codegen-client/src/main/kotlin/software/amazon/smithy/rust/codegen/client/smithy/generators/client/FluentClientGenerator.kt

```diff
@@ -471,7 +473,7 @@ class FluentClientGenerator(
     }
 }
 
-private fun baseClientRuntimePluginsFn(runtimeConfig: RuntimeConfig): RuntimeType =
+private fun baseClientRuntimePluginsFn(runtimeConfig: RuntimeConfig, customizations: List<FluentClientCustomization>): RuntimeType =
```

Comment:
extra arg is unused?

Reply:
I think this was another leftover thing. I've remove the change.